### PR TITLE
Button Component added

### PR DESCRIPTION
**Pull request by PedroAGC**
**Background**

Used to show a tree-structured data.

**Changes done**

- Add Tree Component
- Add Story
- Add Type

### CODE REVIEW

File: src/stories/WButton/WButton.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  if(!props){
+    return null
+  }
+
+  return (
+    <RButton {...props}>{props.text}</RButton>
+  );
+};
+
+export default WButton
```
Comment (christianahvilla) [inline src/stories/WButton/WButton.tsx]:
```suggestion
import Button from 'rsuite/Button';
import { IWButton } from './types';

const WButton = (props: IWButton) => {
  if(!props){
    return null
  }

  return (
    <Button {...props}>{props.text}</Button>
  );
};

export default WButton
```

File: src/stories/WButton/types.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+export interface IWButton {
+  active: boolean;
+  appearance: APPEARENCE;
+  block?: boolean;
+  color?: COLOR;
+  disabled?: boolean;
+  loading?: boolean;
+  size?: SIZE;
+  text: string;
+  onClick?: () => void;
```
Comment (christianahvilla) [inline src/stories/WButton/types.ts]: Sort the interface keys

**Review by christianahvilla — CHANGES_REQUESTED** [review]
Overall looks good, just few comments please fixed before merge this PR

File: src/stories/WButton/WButton.tsx (hunk 1 above)
Comment (PedroAGC) [reply src/stories/WButton/WButton.tsx]: Fixed!

File: src/stories/WButton/types.ts (hunk 2 above)
Comment (PedroAGC) [reply src/stories/WButton/types.ts]: Sorted!

**Review by christianahvilla — APPROVED** [review]
LGTM

**Pull request merged**